config.go

```diff
@@ -975,9 +982,12 @@ func loadConfig() (*config, error) {
 			"minbackoff")
 	}
 
-	// Assert that all worker pools will have a positive number of
-	// workers, otherwise the pools will rendered useless.
-	if err := cfg.Workers.Validate(); err != nil {
+	// Validate the subconfigs for workers and caches.
+	err = lncfg.Validate(
+		cfg.Workers,
```

Comment:
👍

Reply:
hoping that we can eventually move more of our subconfigs into `lncfg` and make use of this pattern, would make `config.go` quite a bit slimmer :)